storybookjs/storybook
# REPLACE babel/runtime with core-js

Issue: https://github.com/storybooks/storybook/issues/4088

## What I did
I removed the alias for babel in the preview webpack config

## How to test
everything should just work
run the fixtures / examples
check on netlify / teamcity
